WebService/WebService/Data/DBContext.cs

```diff
@@ -20,15 +21,6 @@ public DBContext(DbContextOptions<DBContext> options)
 
         protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
-            modelBuilder.Entity<Bar>()
-                .HasMany<SpecificPrice>(b => b.AvailableDrinks)
-                .WithOne(s => s.Bar)
-                .HasForeignKey(s => s.BarID);
-
-            modelBuilder.Entity<Drink>()
-                .HasMany<SpecificPrice>(d => d.DrinkLocations)
-                .WithOne(s => s.Drink)
-                .HasForeignKey(s => s.DrinkID);
```

Comment:
This is used to add migrations to database.

Reply:
Added thsi back in as comment since now they produce error because SpecificPrice no longer contains object references